CMakeLists.txt

```diff
@@ -92,6 +78,7 @@ include(XsdFu)
 include(GTest)
 include(Doxygen)
 include(HeaderTest)
+find_package(OME 5.2.0 REQUIRED Compat Common)
```

Comment:
Good question.  5.2.0 as a minimum is OK for now since it means that 5.1.x won't be allowed, but since the exports aren't provided by 5.1.x anyway, no version is also totally OK.  So it really doesn't make much difference either way at this point.